db/re/item_group_db.yml

```diff
@@ -14208,6 +14208,70 @@ Body:
           - Index: 1
             Item: RWC_Scroll_2012
             UniqueId: true
+  - Group: FINDINGORE
+    SubGroups:
+      - SubGroup: 1
+        List:
+          - Index: 0
+            Item: Emperium
+            Rate: 3
+          - Index: 1
+            Item: Oridecon_Stone
+            Rate: 20
+          - Index: 2
+            Item: Elunium_Stone
+            Rate: 20
+          - Index: 3
+            Item: Gold
+            Rate: 2
+          - Index: 4
+            Item: Oridecon
+            Rate: 10
+          - Index: 5
+            Item: Elunium
+            Rate: 10
+          - Index: 6
+            Item: Boody_Red
+            Rate: 30
+          - Index: 7
+            Item: Crystal_Blue
+            Rate: 30
+          - Index: 8
+            Item: Wind_Of_Verdure
+            Rate: 30
+          - Index: 9
+            Item: Yellow_Live
+            Rate: 30
+          - Index: 10
+            Item: Flame_Heart
+            Rate: 15
+          - Index: 11
+            Item: Mistic_Frozen
+            Rate: 15
+          - Index: 12
+            Item: Rough_Wind
+            Rate: 15
+          - Index: 13
+            Item: Great_Nature
+            Rate: 15
+          - Index: 14
+            Item: Iron
+            Rate: 80
+          - Index: 15
+            Item: Steel
+            Rate: 50
+          - Index: 16
+            Item: Iron_Ore
+            Rate: 100
+          - Index: 17
+            Item: Coal
+            Rate: 60
+          - Index: 18
+            Item: Phracon
+            Rate: 95
+          - Index: 19
+            Item: Emveretarcon
+            Rate: 55
```

Comment:
was removed in this commit. https://github.com/rathena/rathena/commit/97a124ca0e74c590b5efe61777367722ebeba850
if its addition is correct, then it was missing to add it to **item_db.hpp** and **script_constants.hpp**